documents/testing/sections/Coverage.tex

```diff
@@ -0,0 +1,16 @@
+\chapter{Coverage}
+
+Our test harness reaches a total test coverage of 78\%:
+\begin{itemize}
+    \item UI: Coverage 0\% (no automatic tests)
+    \item MORR: Coverage 50\%
+    \item CLI: Coverage 81\%
+    \item KeyboardModule: Coverage 59\%
+    \item WebBrowserModule: Coverage 75\%
+    \item ClipboardModule: Coverage 78\%
+    \item WindowManagementModule: Coverage 85\%
+    \item MouseModule: Coverage 98\%
+    \item Shared: Coverage: 94\%
```

Comment:
You could add the coverage of the WebExtension here as well (85.16% statement coverage according to CI)